src/content/0/en/part0b.md

```diff
@@ -21,7 +21,7 @@ Open the [example application](https://studies.cs.helsinki.fi/exampleapp) on you
 **The 1st rule of web development**: Always keep the Developer Console open on your web browser. On macOS, open the console by pressing `F12` or `option-cmd-i` simultaneously. 
 On Windows or Linux, open the console by pressing `F12` or `ctrl-shift-i` simultaneously. 
 
-Before continuing, find out how to open the Developer Console on your computer (search from Google if necessary) and remember to <i>always</i> keep it open when developing web applications. 
+Before continuing, find out how to open the Developer Console on your computer (Google it if necessary) and remember to <i>always</i> keep it open when developing web applications. 
```

Comment:
I'm pretty sure that to "search from Google" is incorrect. But instead of "Google it" you could probably also write "use Google" or "search with Google".